pyccl/tests/test_hmfunc.py

```diff
@@ -121,17 +121,6 @@ def test_nM_from_string_raises():
         ccl.halos.MassFunc.from_name('Tinker09')
 
 
-def test_nM_default():
-    nM = ccl.halos.MassFunc(COSMO)
-    with pytest.raises(NotImplementedError):
-        nM._get_fsigma(COSMO, 1., 1., 8)
-
-    M_in = 1E12
-    lM_out = nM._get_consistent_mass(COSMO,
-                                     M_in, 1., nM.mdef)
-    assert np.fabs(np.log10(M_in) - lM_out) < 1E-10
-
-
```

Comment:
Why did we drop these tests?

Reply:
Abstract classes can't be instantiated in Python. They will never be instantiated if they are abstract.